go.mod

```diff
@@ -4,20 +4,22 @@ go 1.13
 
 require (
 	github.com/celo-org/kliento v0.1.2-0.20200608140637-c5afc8cf0f44
-	github.com/coinbase/rosetta-sdk-go v0.3.4
-	github.com/ethereum/go-ethereum v1.9.18
+	github.com/coinbase/rosetta-sdk-go v0.5.9
+	github.com/ethereum/go-ethereum v1.9.23
```

Comment:
I'm not sure this change takes any effect because of the replace directive below

Reply:
yeah I let go update this automatically but I think you're right about it not really changing anything